src/main/kotlin/uk/gov/dluhc/printapi/mapper/ElectoralRegistrationOfficeMapper.kt

```diff
@@ -1,18 +1,11 @@
 package uk.gov.dluhc.printapi.mapper
 
 import org.mapstruct.Mapper
-import org.mapstruct.Mapping
 import uk.gov.dluhc.printapi.database.entity.ElectoralRegistrationOffice
-import uk.gov.dluhc.printapi.dto.EroManagementApiEroDto
+import uk.gov.dluhc.printapi.dto.EroContactDetailsDto
 
 @Mapper
 interface ElectoralRegistrationOfficeMapper {
 
-    @Mapping(source = "name", target = "name")
-    @Mapping(target = "phoneNumber", constant = "")
-    @Mapping(target = "emailAddress", constant = "")
-    @Mapping(target = "website", constant = "")
-    @Mapping(target = "address.street", constant = "")
-    @Mapping(target = "address.postcode", constant = "")
-    fun map(dto: EroManagementApiEroDto): ElectoralRegistrationOffice
+    fun map(dto: EroContactDetailsDto): ElectoralRegistrationOffice
```

Comment:
@NathanRussellValtech , seems like we can get rid of this mapper class. 
The `PrintDetailsMapper` will be able map the ero contact details without the help of this class, since there are no exception cases, just mapping fields by name convention.

Reply:
Good idea; done (it also removed a couple of tests with it 👍 )